Src/DependencyCollector/Shared/Implementation/FrameworkHttpProcessing.cs

```diff
@@ -146,6 +146,7 @@ public void OnEndHttpCallback(long id, bool? success, bool synchronous, int? sta
                     // We never collected statusCode or success before 2.1.0-beta4
                     // We also had duplicates if runtime is also 4.5.2 (4.6 runtime has no such problem)
                     // So starting with 2.1.0-beta4 we are cutting support for HTTP dependencies in .NET 4.5.2.
+                    this.TelemetryTable.Remove(id);
```

Comment:
Any side effects of this in addition to the positive effect of letting Event Source to collect request for us?
E.g. any other way to end up with no result code other than the things you mentioned in the issue description?